overlays/nodejs/ec2/placementStrategy.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file contains two categories of exports:
+//
+//     1) A union type, PlacementStrategy, that accepts any valid Placement Strategy type
+//     2) Individual constants for each such Placement Strategy type
+//
+// These give a better developer experience and are just sugared strings.
+
+export let SpreadStrategy: PlacementStrategy = "spread";
```

Comment:
I would encourage adding JSDoc comments on these, since it will get picked up by our doc generation process. Honestly, I usually just copy the existing CloudFormation documentation, since it's usually decent, but in this case I guess we need to go to the CLI docs (since CloudFormation docs aren't very good here).

```
/**
 *  A `spread` placement group places instances on distinct hardware.
 */
```

from https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreatePlacementGroup.html.

Reply:
I've added the comments to the constants and the union, and rebased this on top of `master`.